source-browser.html

```diff
@@ -181,7 +209,46 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           .focus();
       }
 
+      function createNavbar(topic) {
+        $('.li').append(
+          $('<a>', {
+            id: `li${topic}`,
+            href: `#${topic}`
+          }).text(topic)
+        );
+      }
+
+      let list = [];
+      $(document).ready(() => {
```

Comment:
Please don't use multiple DOM ready calls. This can just be added to the one below.

Reply:
I have fixed that.